### PR TITLE
accounts_passwords_pam_tally2: Move to bash_ensure_pam_module_option

### PR DESCRIPTION
**Pull request by dodys**
#### Description:

- As discussed in Gitter, this change removes the degree character and move remediation to new macro `bash_ensure_pam_module_option`.

### CODE REVIEW

**Review by marcusburghardt — COMMENTED** [review]
@dodys , there is also a equivalent macro for Ansible. Do you plan to also update the Ansible remediation?

**Comment by dodys** [comment]
> @dodys , there is also a equivalent macro for Ansible. Do you plan to also update the Ansible remediation?

Pushed a new PR for that, hopefully got it right as I don't have a way to test it right now.

File: linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2/ansible/shared.yml [hunk 1]
```diff
@@ -1,69 +1,11 @@
...
-  shell: grep -e '^\s*account\s\+required\s\+pam_tally2\.so' /etc/pam.d/common-account || true
-  register: check_account_pam_tally2_result
-
-- name: Configure pam_tally2.so module in /etc/pam.d/common-account
-  lineinfile:
-    path: /etc/pam.d/common-account
-    line: 'account required pam_tally2.so'
-    state: present
-  when: '"pam_tally2" not in check_account_pam_tally2_result.stdout'
+{{{ ansible_ensure_pam_module_option('/etc/pam.d/common-auth', 'auth', 'required', 'pam_tally2.so', 'deny', "${var_password_pam_tally2}", '') }}}
```
Comment (marcusburghardt) [inline linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2/ansible/shared.yml:9]:
```suggestion
{{{ ansible_ensure_pam_module_option('/etc/pam.d/common-auth', 'auth', 'required', 'pam_tally2.so', 'deny', "{{ var_password_pam_tally2 }}", '') }}}
```

**Review by marcusburghardt — CHANGES_REQUESTED** [review]
There is just a small syntax issue in the Ansible remediation and it should be fine to be merged.

File: linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2/ansible/shared.yml (hunk 1 above)
Comment (dodys) [reply linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2/ansible/shared.yml]: done! thanks!

**Review by marcusburghardt — APPROVED** [review]

**Pull request merged**